ibc-clients/ics07-tendermint/src/client_state/validation.rs

```diff
@@ -249,11 +249,11 @@ where
         upgrade_path: substitute_upgrade_path,
     } = substitute_client_state;
 
-    (subject_trust_level == substitute_trust_level
-        && subject_unbonding_period == substitute_unbonding_period
-        && subject_max_clock_drift == substitute_max_clock_drift
-        && subject_proof_specs == substitute_proof_specs
-        && subject_upgrade_path == substitute_upgrade_path)
+    (subject_trust_level == &substitute_trust_level
```

Comment:
Good catch on this and the extraneous `clone`!

Reply:
This caught my eye while integrating the client recovery into the Sovereign light client.